### PR TITLE
mgr/dashboard: Exchange PNGs for SVGs

### PR DESCRIPTION
**Pull request by SchoolGuy**
This is kind of a supersede for https://github.com/ceph/ceph/pull/36769

## Checklist
- [x] References tracker ticket - https://tracker.ceph.com/issues/47111
- [x] Updates documentation if necessary
- [x] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     </button>
   </div>
   <div class="modal-body">
-    <img src="assets/Ceph_Logo_Standard_RGB_Reversed_120411_fa.png"
+    <img src="assets/Ceph_Ceph_Logo_with_text_red_white.svg"
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.html:11]: Something is wrong:
image

**Review by tspmelo — CHANGES_REQUESTED** [review]

File: src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.html (hunk 1 above)
Comment (SchoolGuy) [reply src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.html:11]: Fixed through Force Push

**Review by epuertat — COMMENTED** [review]
### Login - Before & After

Bigger sharper logo, but ok.

image


### Navigation - Before & After

Ok here.

image


### About modal - Before & After

In this case the logo looks bigger and the modal is slightly smaller, but no big issue for me. However, there seems to be a color issue, probably related to the SVG color management profile?

#### Chrome (83.0)
image

#### Firefox (79.0)
image

### Red Logo Color mismatch
image

I cannot say which one is Ok:
* `#F1726C` and `#F05C56` are the original Dashboard reds.
* `#EF5B55` is the one from ceph.io as rendered by Chrome :
image
* `#f0424d` is the SVG red.

Summarizing:

image

As this has mainly to do with Ceph branding guidelines, we could perhaps ask for some help from @Thingee here?

**Comment by epuertat** [comment]
BTW @tspmelo would it make sense to enable some form of [visual regression testing](https://www.npmjs.com/package/cypress-visual-regression)?

**Comment by tspmelo** [comment]
@epuertat looks interesting, we could test it.
Wonder how cpu/time consuming it is...

**Comment by epuertat** [comment]
@tspmelo well, it would work for the login page, navigation bar (it seems you can pick specific components), about box, ... and it allows you to provide a threshold of pixels changed, so it's not a hardlimit. If we still find it useful, we could try to CSS hide/make transparent during the e2e the changing text/fields?

**Comment by Thingee** [comment]
@epuertat I'd say go with the color from the SVGs.

File: src/pybind/mgr/dashboard/frontend/src/assets/Ceph_Logo.svg [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      </cc:Work>
+    </rdf:RDF>
+  </metadata>
+  <g
+     transform="translate(-111.75311,-212.54075)"
+     id="layer1"
+     inkscape:groupmode="layer"
+     inkscape:label="Ebene 1">
+    <path
+       style="fill:#f0424d;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:0.264583"
```
Comment (Exotelis) [inline src/pybind/mgr/dashboard/frontend/src/assets/Ceph_Logo.svg:66]: That line could fix the color issue @epuertat mentioned. ef5b55 is the official brand color.

**Review by tspmelo — APPROVED** [review]

**Review by Exotelis — APPROVED** [review]

**Pull request merged**